assets/javascripts/discourse/controllers/admin-plugins-discourse-subscriptions-subscriptions.js.es6

```diff
@@ -1,12 +1,30 @@
+import { alias } from "@ember/object/computed";
+import AdminSubscription from "discourse/plugins/discourse-subscriptions/discourse/models/admin-subscription";
 import Controller from "@ember/controller";
 import showModal from "discourse/lib/show-modal";
 
 export default Controller.extend({
+  loading: false,
+  canLoadMore: alias("model.has_more"),
+
   actions: {
     showCancelModal(subscription) {
       showModal("admin-cancel-subscription", {
         model: subscription,
       });
     },
+
+    loadMore() {
+      if (!this.loading && this.canLoadMore) {
+        this.set("loading", true);
+
+        AdminSubscription.loadMore(this.model.last_record).then((result) => {
```

Comment:
Would be a good idea to `return` this, in case something ever wants to wait for `loadMore`'s promise.